### DISCUSSION TITLE
Can't play YouTube Live Videos

### DISCUSSION THREAD

**Discussion by Marin-Kitagawa** [qutebrowser/qutebrowser#6386] [Q&A] [answered]
Al the YouTube videos are playing in the browser. But I have checked many YT live videos and I can't play any. Is there any special configuration adjustment that must be done or is that a bug? Please check the image below regarding the message displayed by YT while playing the YT live videos.
![YT_Live_Videos_Error](https://user-images.githubusercontent.com/49131888/114399830-fadfe200-9bbe-11eb-966b-a2b82294f97f.png)

**Comment by The-Compiler** [comment, marked as answer]
"YT live videos" as in the live streams on https://www.youtube.com/live I assume? Those work just fine for me. Can you try with `--temp-basedir` and show your `:version` information? How did you install qutebrowser?

**Reply by Marin-Kitagawa** [reply]
Mine is the latest version and I have installed it using the installer for Windows. And `--temp-basedir` still shows the same. Nothing changed and the issue still persists,

The pastebin URL yanked from the version page

[Version Info](https://paste.the-compiler.org/view/922c7842)

**Reply by bitraid** [reply]
Youtube live videos are only available for h264, and this (proprietary) codec is not included in the bundled QT for Windows.

**Reply by SaptakBhoumik** [reply]
You have to compile qt with a propitiatory codecs as it is no included with qt by default
